templates/run_neo4j.java.qute

```diff
@@ -0,0 +1,31 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+//JAVA 10+
```

Comment:
I would use 11 or 17, not the non-lts things…

Reply:
I wanted the minimal required version since I'm using `var` 
Happy to bump to a LTS though